src/databricks/labs/ucx/install.py

```diff
@@ -399,13 +386,13 @@ def __init__(  # pylint: disable=too-many-arguments
         skip_dashboards=False,
     ):
         self._config = config
-        self._installation = installation
```

Comment:
you don't extend the `WorkspaceContext`, so you can't change the visibility of properties in `WorkspaceInstallation`.

Reply:
yes, overzealous find & replace